src/components/common/modal/CashPayFailModal.vue

```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+import DanjiButton from '../button/DanjiButton.vue'
+import { useRouter } from 'vue-router'
+import fail from '@/assets/icons/fail.svg'
+
+const router = useRouter()
+
+const goToChargePage = () => {
+  router.push('/') // 임시로 홈 이동 설정(나중에 아마도 변경 예정)
+}
+</script>
+
+<template>
+  <div class="fixed inset-0 z-[1000] flex items-center justify-center">
+    <!-- 오버레이 -->
+    <div class="absolute inset-0 bg-black bg-opacity-50 z-[1000]" />
```

Comment:
색상 코드는 똑같긴 한데, 그래도 저희 컬러 시스템에서 뽑아서 쓰는 건 어떤가요?!
```suggestion
    <div class="absolute inset-0 bg-Black-1 bg-opacity-50 z-[1000]" />
```